libs/core/src/components/sage-tabpanel/sage-tabpanel.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, Element, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-tabpanel',
+  styleUrl: 'sage-tabpanel.scss',
+  shadow: false,
+})
+export class SageTabpanel {
+  @Element() el: HTMLElement;
+
+  /**
+   * Sets the related tab name, this name must match a `sage-tab`'s tab property
+  */
+  @Prop() tab!: string;
```

Comment:
possibly a rename to `tabName` or `tabname` if we don't want to deal with the hyphen at the element attribute level.

Reply:
I've updated it to `name`, that should work fine and is in alignment with HTML attribute naming.